### PR TITLE
Add StyleCop to all the projects.

### PR DESCRIPTION
**Pull request by dlemstra**

### CODE REVIEW

File: ImgBot.Test/ImgBot.Test.csproj [hunk 1]
```diff
@@ -1,111 +1,51 @@
...
-      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
-    </Content>
-    <Content Include="data\folder\item3.jpg">
-      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
-    </Content>
-  </ItemGroup>
-  <Import Project="$(VSToolsPath)\TeamTest\Microsoft.TestTools.targets" Condition="Exists('$(VSToolsPath)\TeamTest\Microsoft.TestTools.targets')" />
-  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
-  <Target Name="EnsureNuGetPackageBuildImports" BeforeTargets="PrepareForBuild">
+﻿<Project Sdk="Microsoft.NET.Sdk">
```
Comment (dabutvin) [inline ImgBot.Test/ImgBot.Test.csproj]: 👍

Comment (dabutvin) [inline ImgBot.Test/ImgBot.Test.csproj]: 👍

**Review by dabutvin — APPROVED** [review]

**Review by dabutvin — APPROVED** [review]

**Comment by dlemstra** [comment]
You want me to merge this into master and then merge master into your PR to fix the conflicts there?

**Comment by dabutvin** [comment]
go for it - I can fix up my other branch to get used to the style settings

**Pull request merged**